### ISSUE TITLE
Add configuration to skip tests

### ISSUE THREAD

**Issue by brynrhodes** [cqframework/cql-tests-runner#8]
There should be a "skip" input parameter that takes a list of names of tests to be skipped.

**Comment by darkknight13** [comment]
Will there be an option to skip by tests, by group, or by a test only? If we intend to support skipping by tests and by group, in addition to skipping a test, should we use the same parameter "skip" or separate parameters such as "skip-tests", "skip-group", and "skip-test"? Alternatively, another option could be to allow anything to be passed to the "skip" parameter—tests, groups, or test names—and if it matches anything, it will be skipped.

**Comment by richfirely** [comment]
I added some functionality like this in the linked PR. I had to get past some tests during development of said PR. Hopefully it's easily adjusted to account for ideas here.  For the moment it takes all 3, tests, group, test.

**Comment by brynrhodes** [comment]
Yeah, looks like in order for this to work, the schema validation needs to be relaxed so that a skip can specify only a tests name or only a group name. Right now if you try these configs it gets a validation error on the config and won't run.

**Comment by cmoesel** [comment]
It would be nice to have this feature for the new `OnlyList` feature I added last week too.
